src/main/java/org/jfrog/hudson/ivy/ArtifactoryIvyRunListener.java

```diff
@@ -56,12 +56,10 @@ public void onCompleted(AbstractBuild run, TaskListener listener) {
                 return;
             }
             if (artifactoryIvyConfigurator.isDeployBuildInfo()) {
-                String buildName = BuildUniqueIdentifierHelper.getBuildNameConsiderOverride(artifactoryIvyConfigurator, run);
+                String buildName = BuildUniqueIdentifierHelper.getBuildNameConsiderOverride(artifactoryIvyConfigurator, run, null);
```

Comment:
Any reason why we sent null here as an argument, instead of env? Would the following work to get the env?
```
EnvVars env = run.getEnvironment(listener);
```